gobblin-core/src/main/java/gobblin/source/extractor/filebased/FileBasedExtractor.java

```diff
@@ -174,12 +186,12 @@ public long getHighWatermark() {
    */
   @SuppressWarnings("unchecked")
   public Iterator<D> downloadFile(String file) throws IOException {
-    LOGGER.info("Beginning to download file: " + file);
+    LOG.info("Beginning to download file: " + file);
 
     try {
       InputStream inputStream = this.closer.register(this.fsHelper.getFileStream(file));
       Iterator<D> fileItr = (Iterator<D>) IOUtils.lineIterator(inputStream, ConfigurationKeys.DEFAULT_CHARSET_ENCODING);
-      if (workUnitState.getPropAsBoolean(ConfigurationKeys.SOURCE_SKIP_FIRST_RECORD, false) && fileItr.hasNext()) {
+      if (this.workUnitState.getPropAsBoolean(ConfigurationKeys.SOURCE_SKIP_FIRST_RECORD, false) && fileItr.hasNext()) {
```

Comment:
I believe the use case was for ingestion of csv data from certain sources. Some sources have files where the first line of the file is a list of the column names in the file, and then each subsequent row would be an actual record.